pkg/service/replay/utils.go

```diff
@@ -66,7 +69,31 @@ func (t *testUtils) SimulateRequest(ctx context.Context, _ uint64, tc *models.Te
 	return nil, nil
 }
 
-func (t *testUtils) AfterTestHook(_ context.Context, testRunID, testSetID string, tsCnt int) (*models.TestReport, error) {
+func (t *requestMockUtil) AfterTestHook(_ context.Context, testRunID, testSetID string, tsCnt int) (*models.TestReport, error) {
 	t.logger.Debug("AfterTestHook", zap.Any("testRunID", testRunID), zap.Any("testSetID", testSetID), zap.Any("totTestSetCount", tsCnt))
 	return nil, nil
 }
+
+func (t *requestMockUtil) ProcessTestRunStatus(ctx context.Context, status bool, testSetID string) {
+	if err := ctx.Err(); err != nil {
```

Comment:
you can replace ctx in the function with underscore rather than this check for avoding linter issue. same with all other functions.

Reply:
have done so